src/styles.css

```
@@ -186,6 +186,10 @@ body{background:var(--bg);color:var(--ink);font-family:'Plus Jakarta Sans',sans-
 .bna{background:var(--bg);color:var(--muted);border-color:var(--border2)}
 .bgfi{background:rgba(26,86,219,.07);color:var(--blue);border-color:rgba(26,86,219,.2);cursor:default}
 .tags{display:flex;flex-wrap:wrap;gap:4px}
+.bwinner{background:#FFFAED;color:#92600A;border-color:rgba(251,191,36,.3);cursor:pointer;transition:all .18s;text-decoration:none}
```

Stylelint (17.10.0)

[error] Expected "rgba" to be "rgb" (color-function-alias-notation)

(color-function-alias-notation)